snapcraft/internal/project_loader.py

```diff
@@ -175,6 +176,42 @@ def snap_env(self):
 
         return env
 
+    def project_env(self):
+        return [
+            'SNAPCRAFT_STAGE={}'.format(self._project_options.stage_dir),
+            'SNAPCRAFT_PROJECT_NAME={}'.format(self.data['name']),
+            'SNAPCRAFT_PROJECT_VERSION={}'.format(self.data['version']),
+        ]
+
+    def _expand_env(self, snapcraft_yaml):
+        skip_keys = ['name', 'version']
```

Comment:
Yeah, I figured. Man is that ever the corner of cases :stuck_out_tongue: . But valid nonetheless.